### ISSUE TITLE
[$500] Web - The image overflows from the container of a request with scan

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#27433]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Start chat
2. Click plus sign
3. Select Request money
4-Click in Scan and click in choose file
5-Select a picture  and click Request button

## Expected Result:
 It should correctly display the image in the container.

## Actual Result:
Image overflow into the container.

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.70.2
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/93399543/24ee3d57-468b-4f7b-8f09-b11e5bfbb8b5

https://github.com/Expensify/App/assets/93399543/363450f1-3319-48ba-89b1-2aff0ddd4436

**Expensify/Expensify Issue URL:**
**Issue reported by:** @Mauri-salazar
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1694443199222289

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01a96497515ab13e52</li>
        <li>Upwork Job ID: 1702298845230567424</li>
        <li>Last Price Increase: 2023-09-28</li>
    </ul>
</details>

**Comment by zukilover** [comment]
## Proposal
### Please re-state the problem that we are trying to solve in this issue.
The image overflows from the container of a request with scan

### What is the root cause of that problem?
This happens because the image is resized with `cover` mode. This caused small images seem to be stretched to the container.

https://github.com/Expensify/App/blob/95c9e4743f44ca42ece2c6ac56259b911d5feafc/src/components/ImageWithSizeCalculation.js#L77-L78

### What changes do you think we should make in order to solve the problem?
Change the resize mode to `contain`.

```
             resizeMode={Image.resizeMode.contain}
```

### What alternative solutions did you explore? (Optional)
N/A

**Comment by MitchExpensify** [comment]
Checking on the project curator if this is known https://expensify.slack.com/archives/C049HHMV9SM/p1694799296149659?thread_ts=1694443199.222289&cid=C049HHMV9SM

**Comment by MitchExpensify** [comment]
Waiting on proposals here

**Comment by MitchExpensify** [comment]
Still waitin'

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (First week)

**Comment by MitchExpensify** [comment]
Makes sense @Krishna2323 , I'll keep this open until that hit staging to confirm its fixed. Although @mvtglobally are you saying you cannot reproduce this now? Maybe we can close now versus putting on hold 🤔

**Comment by MitchExpensify** [comment]
Nor reproducible, closing
